web_ui/src/webdata.ts

```diff
@@ -1,4 +1,5 @@
 export type WebData<T> =
+  | { readonly status: "initial" }
```

Comment:
This is necessary for when we have api calls we don't want to call immediately, like mutations.